### ISSUE TITLE
If you host the site, but use DigitalOcean or AWS, who's what?

### ISSUE THREAD

**Issue by andrewfairlie** [madebymutual/GDPR-ePR-Guide#2]
So say the client who owns the website is the data-controller, and they hire you to host and maintain their site. You're a data-processor, right?

But you use a third party like DigitalOcean or AWS to host a VPN for it. They're also a data-processor, I presume.

So in that case, what's the relationship there?

Is it.

1. The client has separate data processing agreements with both you and DigitalOcean
2. The client has a data processing agreement with you - you have a data processing agreement with DigitalOcean where you act as the controller?

**Comment by croxton** [comment]
The answer I think would have to be (2): your client has a contract and data-processing agreement with you (the website developer), you have a contract and data-processing agreement with the host. 

It does rather leave the agency in vulnerable position in the event of a data breach. If your client can prove that you did not meet the terms of the data-processing agreement then they could sue you to recover any losses they may incur.

We usually include a clause in our contracts that states:

> You are solely responsible for any data uploaded to your web server. You must ensure that uploaded data complies with the relevant laws in the countries that it can be accessed. [Agency name] does not make any warranties or representations regarding any data and/or information that is uploaded to your web server.
> 
> Under no circumstances will [Agency name] be liable for the loss, theft or inadvertent exposure of your data, either as a result of hardware failure, software failure or any other cause.

**Comment by andrewfairlie** [comment]
Yes very good idea. We have a similar clause that the company is never liable for more than the total cost of the project on launch date and we have suitable indemnity insurance to cover that.
